src/utils.js

```diff
@@ -12,18 +12,46 @@ export const notify = (title, description) => {
  * @return {string} A link to a default profile picture.
  */
 export const generateDefaultProfile = () => {
-  const eyes = ['eyes1', 'eyes10', 'eyes2', 'eyes3', 'eyes4', 'eyes5', 'eyes6', 'eyes7', 'eyes9'];
-  const nose = ['nose2', 'nose3', 'nose4', 'nose5', 'nose6', 'nose7', 'nose8', 'nose9'];
-  const mouth = ['mouth1', 'mouth10', 'mouth11', 'mouth3', 'mouth5', 'mouth6', 'mouth7', 'mouth9'];
+  const eyes = [
+    'eyes1',
+    'eyes10',
+    'eyes2',
+    'eyes3',
+    'eyes4',
+    'eyes5',
+    'eyes6',
+    'eyes7',
+    'eyes9',
+  ];
+  const nose = [
+    'nose2',
+    'nose3',
+    'nose4',
+    'nose5',
+    'nose6',
+    'nose7',
+    'nose8',
+    'nose9',
+  ];
+  const mouth = [
+    'mouth1',
+    'mouth10',
+    'mouth11',
+    'mouth3',
+    'mouth5',
+    'mouth6',
+    'mouth7',
+    'mouth9',
+  ];
   const colors = ['22ACEA', '0659BC', 'BED9E6'];
 
-  const getRandom = list => {
+  const getRandom = (list) => {
     return list[Math.floor(Math.random() * list.length)];
   };
 
-  return `https://api.adorable.io/avatars/face/${getRandom(eyes)}/${getRandom(nose)}/${getRandom(mouth)}/${getRandom(
-    colors
-  )}`;
+  return `https://api.adorable.io/avatars/face/${getRandom(eyes)}/${getRandom(
+    nose
```

Comment:
airbnb lint makes you do this?? wierd line break

Reply:
Hmmm, I thought I got rid of the line limit. I'll try switching it back and see what it says.

Reply:
It's showing it as a prettier error. I'll split it at a cleaner spot though.